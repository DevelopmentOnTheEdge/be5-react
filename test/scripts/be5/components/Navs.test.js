import React          from 'react';
import renderer       from 'react-test-renderer';
import Navs     from '../../../../src/scripts/be5/components/Navs';


test('test', () => {
  const steps = [
    {title: "Companies", url: '#!text/A'},
    {title: "EgissoDocs", url: '#!text/B'},
  ];

  const component = renderer.create(
      <Navs steps={steps} tabs />
  );

  expect(component.toJSON()).toMatchSnapshot();
});
