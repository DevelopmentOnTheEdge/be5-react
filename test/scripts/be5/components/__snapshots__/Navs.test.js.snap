// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `
<div
  className="navs-component"
>
  <ul
    className="nav nav-tabs"
  >
    <li
      className="nav-item"
    >
      <a
        className="nav-link active"
        href="#"
        onClick={[Function]}
      >
        Companies
      </a>
    </li>
    <li
      className="nav-item"
    >
      <a
        className="nav-link"
        href="#"
        onClick={[Function]}
      >
        EgissoDocs
      </a>
    </li>
  </ul>
  <div
    className="tab-content"
  >
    <div
      className="document-content"
      id="document-content___navs"
    >
      <h1>
        A
      </h1>
    </div>
  </div>
</div>
`;
